packages/components/src/DataList/DataList.const.ts

```diff
@@ -20,3 +20,5 @@ export const BREAKPOINT_SIZES: Record<Breakpoints, number> = {
   lg: 1080,
   xl: 1440,
 };
+
+export const SEARCH_DEBOUNCE_DELAY = 500;
```

Comment:
```suggestion
export const SEARCH_DEBOUNCE_DELAY = tokens["timing-slowest"];
```

What do you think? Is it the same type of timing?

Reply:
it can yeah, i believe i used it in one of the files here